string wheel
float32 current_amps
float32 vel_m_s
